app/components/layer-file-settings/component.js

```diff
@@ -1,4 +1,15 @@
 import Ember from 'ember';
 
 export default Ember.Component.extend({
+	file_object: null,
+	actions: {
+       fileDetail(file) {
+       		this.set('layer.settings.properties.downloadLink' , file._internalModel.__data.links.download)
```

Comment:
Is this right? ```file._internalModel.__data.links.download``` Looks like very weird structure. Is there not another way to access the download link?